src/SSHDebugPS/SSHDebugPS.csproj

```diff
@@ -55,13 +55,18 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
   <ItemGroup>
-    <None Include="project.json" />
     <Reference Include="liblinux">
       <HintPath>$(GeneratedAssembliesDir)liblinux.dll</HintPath>
     </Reference>
     <Reference Include="Microsoft.VisualStudio.Debugger.InteropA, Version=9.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
       <Private>False</Private>
     </Reference>
+    <Reference Include="Microsoft.VisualStudio.ImageCatalog, Version=15.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
+      <HintPath>$(NuGetPackagesDirectory)\Microsoft.VisualStudio.ImageCatalog.$(Microsoft_VisualStudio_Shell_15_0_Version)\lib\net45\Microsoft.VisualStudio.ImageCatalog.dll</HintPath>
```

Comment:
>HintPath [](start = 7, length = 8)

Should all of these have `<Private>False</Private>` to avoid msbuild's copy-local behavior?

Reply:
I added it for the two I added and for the Linux ConnectionManager one. 

---
In reply to: [287510686](https://github.com/microsoft/MIEngine/pull/868#discussion_r287510686) [](ancestors = 287510686)